pkg/discovery/worker/dispatcher.go

```diff
@@ -81,7 +81,15 @@ func (d *Dispatcher) Dispatch(nodes []*api.Node, cluster *repository.ClusterSumm
 	for assignedNodesCount+perTaskNodeLength <= len(nodes) {
 		currNodes := nodes[assignedNodesCount : assignedNodesCount+perTaskNodeLength]
 
-		currPods := d.config.clusterInfoScraper.GetRunningAndReadyPodsOnNodes(currNodes)
+		var currPods []*api.Pod
+		//currPods := d.config.clusterInfoScraper.GetRunningAndReadyPodsOnNodes(currNodes)
```

Comment:
Nit: Any reason of retaining this ? If not please remove (here an at all places below).